pygmt/modules.py

```diff
@@ -12,6 +12,26 @@
 )
 from .exceptions import GMTInvalidInput
 
+SPECIAL_PARAMS = {
+    "FONT": [
+        "FONT_ANNOT_PRIMARY",
+        "FONT_ANNOT_SECONDARY",
+        "FONT_HEADING",
+        "FONT_LABEL",
+        "FONT_TAG",
+        "FONT_TITLE",
+    ],
+    "FONT_ANNOT": ["FONT_ANNOT_PRIMARY", "FONT_ANNOT_SECONDARY"],
+    "FORMAT_TIME_MAP": ["FORMAT_TIME_PRIMARY_MAP", "FORMAT_TIME_SECONDARY_MAP"],
+    "MAP_ANNOT_OFFSET": ["MAP_ANNOT_OFFSET_PRIMARY", "MAP_ANNOT_OFFSET_SECONDARY"],
+    "MAP_GRID_CROSS_SIZE": [
+        "MAP_GRID_CROSS_SIZE_PRIMARY",
+        "MAP_GRID_CROSS_SIZE_SECONDARY",
+    ],
```

Comment:
Please also add **MAP_GRID_PEN**, which is undocumented (see https://github.com/GenericMappingTools/gmt/pull/3318).

Reply:
Done.

![MAP_GRID_PEN config change](https://raw.githubusercontent.com/GenericMappingTools/pygmt/9e85bbe422a38a3f9e3b322e35f7e12162244fd3/pygmt/tests/baseline/test_config_map_grid_pen.png)